tests/serverpod_test_server/test_integration/test_tools/serverpod_test_tools.dart

```diff
@@ -346,10 +346,10 @@ class _AsyncTasksEndpoint {
         createSessionCallback: (_) => _localUniqueSession,
         endpointPath: 'asyncTasks',
         methodName: 'insertRowToSimpleDataAfterDelay',
-        parameters: _i1.testObjectToJson({
-          'num': num,
-          'seconds': seconds,
-        }),
+        parameters: {
+          'num': _i1.testObjectToJson(num),
+          'seconds': _i1.testObjectToJson(seconds),
```

Comment:
Question: The first commit has these changes. This should not be part of this PR anymore right?

Reply:
I should have dropped that commit before opening the PR, my mistake. It's not a technical issue though since the next `build` corrects this (it was rebased on main).